_posts/en/newsletters/2021-12-22-newsletter.md

```diff
@@ -0,0 +1,665 @@
+pay for and receive the incoming capacity using a [dual funded][topic
+dual funding] channel open. Although there’s no way to enforce that the
+advertising node actually routes payments, the proposal does incorporate
+an earlier proposal also [later used][lnd#5709] in Lightning Pool that
+prevents the advertiser from using their money for other purposes until
+the agreed upon lease period has concluded.  That means refusing to
+route payments would provide no advantages but would deny the
+advertising node the opportunity to earn routing fees.
+
+<!-- TODO:EXTRA:table comparing liquidity ads and Pool -->
+
+Three years after [first being proposed][descriptor gist] for Bitcoin
+Core, [draft BIPs][descriptor bips1] were [created][descriptor bips2] for
+[output script descriptors][topic descriptors].  Descriptors are strings
+that contain all the information necessary to allow a wallet or other
+program to track payments made to or spent from a particular script or
+set of related scripts (i.e. an address or a set of related addresses
+such as in an HD wallet).  Descriptors combine well with [miniscript][topic miniscript] in
+allowing a wallet to handle tracking and signing for a larger variety of
+scripts.  They also combine well with [PSBTs][topic psbt] in allowing
+the wallet to determine which keys it controls in a multisig script.  By
+the end of the year, Bitcoin Core had made descriptor-based wallets its
+[default][descriptor default] for newly-created wallets.
+
+A common way of opening LN channels that had never before been official
+supported began to be [specified][0conf channels] in July.  Zero-conf
+channel opens, also called *turbo channels*, are new single-funded
+channels where the funder gives some or all of their initial funds to
+the acceptor. Those funds are not secure until the channel open
+transaction receives a sufficient number of confirmations, so there’s no
+risk to the acceptor spending some of those funds back through the
+funder using the standard LN protocol.  For example, Alice has several
+BTC in an account at Bob’s custodial exchange. Alice asks Bob to open a
+new channel paying her 1.0 BTC. Because Bob trusts himself not to
+double-spend the channel he just opened, he can allow Alice to send 0.1
+BTC through his node to third-party Carol even before the channel open
+transaction has received a single confirmation.  Specifying the behavior
+should help improve interoperability between LN nodes and the merchants
+who offer this service.
+
+{:.center}
+![Zero-conf channel illustration](/img/posts/2021-07-zeroconf-channels.png)
+
+Two related proposals for new signature hash (sighash) types were
+[combined][sighash combo] into [BIP118][].  `SIGHASH_NOINPUT`, proposed
+in 2017 and partly based on previous proposals going back a decade, was
+superseded by [`SIGHASH_ANYPREVOUT`][topic sighash_anyprevout] first
+[proposed][anyprevout proposed] in 2019.  The new sighash types will
```

Comment:
I think mentioning `SIGHASH_ANYPREVOUTANYSCRIPT` in some capacity before "the new sighash typeS" would help with clarity.